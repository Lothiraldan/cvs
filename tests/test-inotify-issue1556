#!/bin/sh

"$TESTDIR/hghave" inotify || exit 80

hg init

touch a b
hg add a b
rm b

echo % status without inotify
hg st

echo "[extensions]" >> $HGRCPATH
echo "inotify=" >> $HGRCPATH

echo % inserve
hg inserve -d --pid-file=hg.pid 2>&1
cat hg.pid >> "$DAEMON_PIDS"

echo % status
hg st

sleep 1
echo "Are we able to kill the service? if not, the service died on some error"
kill `cat hg.pid`

