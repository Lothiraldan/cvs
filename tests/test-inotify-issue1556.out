% status without inotify
A a
! b
% inserve
% status
A a
! b
? hg.pid
Are we able to kill the service? if not, the service died on some error
